internal/ingress/controller/template/template.go

```diff
@@ -877,14 +877,34 @@ func buildRateLimit(input interface{}) []string {
 	}
 
 	if loc.RateLimit.RPS.Limit > 0 {
-		limit := fmt.Sprintf("limit_req zone=%v burst=%v nodelay;",
-			loc.RateLimit.RPS.Name, loc.RateLimit.RPS.Burst)
+		limit := fmt.Sprintf("limit_req zone=%v", loc.RateLimit.RPS.Name)
```

Comment:
please use %d instead of %v on any call. or %s. In this case, as this is a string, let's stick with %s and on the other cases with %d